Source/WebCore/loader/PingLoader.cpp

```diff
@@ -223,7 +223,8 @@ void PingLoader::startPingLoad(LocalFrame& frame, ResourceRequest& request, HTTP
     // Document in the Frame, but the activeDocumentLoader will be associated
     // with the provisional DocumentLoader if there is a provisional
     // DocumentLoader.
-    bool shouldUseCredentialStorage = frame.protectedLoader()->client().shouldUseCredentialStorage(frame.loader().activeDocumentLoader(), identifier);
+    RefPtr activeDocumentLoader = frame.loader().activeDocumentLoader();
```

Comment:
We don't need a local variable here, should just use `protectedActiveDocumentLoader()`